packages/salesforcedx-slds-linter/package-lock.json

```diff
@@ -0,0 +1,4740 @@
+{
```

Comment:
Could you remove this file? Let's not commit the package-lock.json files for now.